snapshots/types/format.go

```diff
@@ -3,4 +3,4 @@ package types
 // CurrentFormat is the currently used format for snapshots. Snapshots using the same format
 // must be identical across all nodes for a given height, so this must be bumped when the binary
 // snapshot output changes.
-const CurrentFormat uint32 = 2
+const CurrentFormat uint32 = 3
```

Comment:
I am curious if we need to change this or not.

Reply:
might not, since snapshot taken under 1.18 or 1.19 can be restore correctly by receiver on either version